backend/.env.example

```diff
@@ -1 +1 @@
-DATABASE_URL=""
\ No newline at end of file
+DATABASE_URL=""
```

Comment:
Since there is no change in this file, the PR shouldn't have shown it here. 
`git checkout master -- ./backend/.env.example` should revert the code to master.